csvs/english/set.csv

```diff
@@ -33,3 +33,4 @@ pfLLLbcmBpfkqrR9fKGtj	EVR	Everfest	F	KQGqrFhCnDmkNFcbLBNhp - F	2022-02-04T00:00:
 dBgRbDmcFzW9kGnPFrqQ8	UPR	Uprising	N	RFJzQMnLBzFNqjL7BdJgj - N	2022-06-24T00:00:00.000Z	null	UPR000	UPR225	https://fabtcg.com/products/booster-set/uprising/	https://fabtcg.com/collectors-centre/uprising/	https://fabtcg.com/resources/card-galleries/uprising/
 HWKQRtfbPPRGqh6dMdQFK	DYN	Dynasty	N	9Crgw7PnFN69zb9DKzFTb - N	2022-11-11T00:00:00.000Z	null	DYN000	DYN246	https://fabtcg.com/products/booster-set/dynasty/	https://fabtcg.com/collectors-centre/dynasty/	https://fabtcg.com/resources/card-galleries/dynasty/
 rqprmGL98FMqcPzPBpKmQ	OUT	Outsiders	N	zd6wBL7zcQR7zrJhwCjrh - N	2023-03-24T00:00:00.000Z	null	OUT000	OUT238	null	https://fabtcg.com/collectors-centre/outsiders/	https://fabtcg.com/resources/card-galleries/outsiders/
+7LbfKJbkWGGwFJDWJn8GB	ARA							ARA029	null	null	null
```

Comment:
The ARA set seems to  be missing values for name, edition, edition unique ids, initial release dates, out of print dates, and start ID!

Reply:
Where does edition unique ids come from?

Reply:
I added the missing data and reran the precommit scripts, it seems to have added the missing identifier.

The generate-json script is failing and I'm not sure why or on which card, makes me think something went wrong with my data entry